10.1.0/startup/generateStartupCommand.js

```diff
@@ -61,18 +61,12 @@ if (!startupCommand && !nodeFile) {
     nodeFile = DEFAULTAPP;
 }
 
-if (!startupCommand && nodeFile && fs.existsSync(nodeFile)) {
-    if (process.env.APPSVC_REMOTE_DEBUGGING == "TRUE") {
```

Comment:
why is this removed?

Reply:
if the startupCmd is process.json, the startupCommand will be `node process.json` which doesnt work
this call has to handle .js files .json files

pm2 should start the node-wrapper which has logic to insert --inspect

I'll double check to make sure it works